apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.controller.ts

```diff
@@ -120,11 +120,15 @@ export class LimitedAccessCaseController {
     JwtAuthUserGuard,
     RolesGuard,
     LimitedAccessCaseExistsGuard,
-    new CaseTypeGuard([...restrictionCases, ...investigationCases]),
+    new CaseTypeGuard([
+      ...restrictionCases,
+      ...investigationCases,
+      ...indictmentCases,
+    ]),
     CaseWriteGuard,
     CaseCompletedGuard,
   )
-  @RolesRules(defenderUpdateRule)
+  @RolesRules(prisonSystemStaffRule, defenderUpdateRule)
```

Comment:
This will allow prison staff, both admin and prison, to update all fields in all cases.

Instead of using `prisonSystemStaffRule`, you need to create a separate update rule for prison admin staff, similar to the `defenderUpdateRule`. Note that the rules for defenders and prison admin staff need to give access to different fields.

Reply:
I changed it! 🤓